tests/sles4sap/hana_test.pm

```diff
@@ -22,18 +22,15 @@ sub test_python3 {
     # The following commands only make sense on a cluster
     return unless get_var('CLUSTER_NAME');
```

Comment:
With the change below with `-m py_compile` is the "run only on a cluster" restriction still valid?

Reply:
> With the change below with `-m py_compile` is the "run only on a cluster" restriction still valid?

Yes. We only need to check that the scripts are in the Python 3 dialect.

Reply:
Yes, but why not run the test in cluster and non-cluster scenarios? It made sense before as you were running the python scripts, but now you're only checking them with `-m py_compile`.